extensions/sql-migration/src/wizard/wizardController.ts

```diff
@@ -4,6 +4,7 @@
  *--------------------------------------------------------------------------------------------*/
 import * as azdata from 'azdata';
 import * as vscode from 'vscode';
+import * as mssql from '../../../mssql';
```

Comment:
We should expose this in a different way, I'll work on that once the PR is merged. Not a fan of doing cross extension imports.

Reply:
sounds good.  let's change that in the next PR. 😄